app/datasources/db/models.py

```diff
@@ -120,6 +133,21 @@ async def get_abis_sorted_by_relevance(cls) -> AsyncIterator[ABI]:
         for result in results.scalars().all():
             yield cast(ABI, result)
 
+    @classmethod
+    async def get_abi_json_newer_equal_than(
```

Comment:
```suggestion
    async def get_abi_newer_equal_than_creation_date(
```
I would like to remove json to be aligned with `get_abis_sorted_by_relevance`. Also specify that is comparing with `created` date and not other dates that are in the ABI model.

Reply:
I prefer to get only the JSON, as I don't need the other information. It's a operation we will do a lot and I want to optimize it as much as we can.

But agree on documenting better, done